ElrondNetwork/api.elrond.com
# Logo resolver for tokens and collections

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- No endpoint that redirects to token/collection logo png or svg content
  
## Proposed Changes
- Add endpoints that redirects to token/collection logo

## How to test
- Check if /logo/:identifier/png with a valid token or collection identifier redirects to png logo
- Check same endpoint with a invalid token or collection identifier
